Backend/WebApi-PointMap/Controllers/ApplicationLaunchFromSS0Controller.cs

```diff
@@ -0,0 +1,49 @@
+﻿using DataAccessLayer.Database;
+using ManagerLayer.AccessControl;
+using ManagerLayer.Login;
+using ManagerLayer.UserManagement;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using WebApi_PointMap.Models;
+
+namespace WebApi_PointMap.Controllers
+{
+    public class ApplicationLaunchFromSS0Controller : ApiController
+    {
+        UserLoginManager _userLoginManager;
+
+        // POST api/ssolaunch/user/login
+        [HttpPost]
+        [Route("api/sso/user/login")]
+        public IHttpActionResult LoginFromSSO([FromBody] LoginPOCO value) //using a POCO to represent request
+        {
+            _userLoginManager = new UserLoginManager();
+            ResponsePOCO response;
+            if (value == null)
+            {
+                response = new ResponsePOCO { Data = null, Timestamp = DateTime.UtcNow };
+                return Ok(response);
+            }
+            var loginAttempt = _userLoginManager.LoginFromSSO(value.Email, value.SSOUserId);

```

Comment:
The email may (and will) change. Use the SSOUserId ONLY.

Reply:
How will SSOuserid be used/stored in the individual application? will it take place of the user username?